File: app/src/main/java/org/mifos/mobile/ui/views/RegistrationView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by dilpreet on 31/7/17.
 */

public interface RegistrationView extends MVPView {

    void showRegisteredSuccessfully();

    void showError(String msg);
}

File: app/src/main/java/org/mifos/mobile/ui/views/RegistrationView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 31/7/17.
 */
interface RegistrationView : MVPView {
    fun showRegisteredSuccessfully()
    fun showError(msg: String?)
}